README.md

```diff
@@ -175,6 +185,68 @@ jobs:
           body: ${{ steps.calibre.outputs.markdown }}
 ```
 
+## Combined workflow
+
+You can combine all of the above into one all-encompassing workflow
+
+```yml
+# Compress images:
+# - on pull requests from this report containing images
+# - on pushing of images to master (for forks)
+# - on demand
+# - at 11pm every Sunday just in case anything gets missed with any of the above
+# For pull requests, the images are added to the PR
+# For the rest a new PR is opened if any images are compressed.
+name: Compress images
+on:
+  pull_request:
+    paths:
+      - '**.jpg'
+      - '**.jpeg'
+      - '**.png'
+      - '**.webp'
+  push:
+    branches:
+      - master
+    paths:
+      - '**.jpg'
+      - '**.jpeg'
+      - '**.png'
+      - '**.webp'
+  workflow_dispatch:
+  schedule:
+    - cron: '00 23 * * 0'
+jobs:
+  build:
+    name: calibreapp/image-actions
+    runs-on: ubuntu-latest
+    # Only run on main repo on and PRs that match the main repo
+    if: |
+      github.repository == 'example/example_repo' &&
+      (github.event_name != 'pull_request' ||
+       github.event.pull_request.head.repo.full_name == github.repository)
+    steps:
+      - name: Checkout Branch
+        uses: actions/checkout@master
+      - name: Compress Images
+        id: calibre
+        uses: calibreapp/image-actions@master
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          compressOnly: ${{ github.event_name != 'pull_request' }}

```

Comment:
Could you add a comment above this to explain how/why it works?

Reply:
Changed it to this:

```yml
          # For non-pull requests, run in compressOnly mode and we'll PR after
          compressOnly: ${{ github.event_name != 'pull_request' }}
```